package.json

```diff
@@ -9,25 +9,50 @@
     "open data",
     "open knowledge"
   ],
-  "main": "index.js",
+  "main": "lib/index.js",
   "dependencies": {
-    "bluebird": "^2.9.30",
     "csv": "^0.4.2",
```

Comment:
@zhenyab please check, as I do not think csv is actually a dependency. it might be used in the tests only. Also, we should rely on `csv-parse` which is just the parsing component we probably use in tests. I know this is old code, but good to clean up